modules/nw-configuring-elb-timeouts-aws-classic.adoc

```diff
@@ -0,0 +1,11 @@
+// Module included in the following assemblies:
+//
+// * networking/configuring_ingress_cluster_traffic/configuring-ingress-cluster-traffic-aws.adoc
+
+:_content-type: CONCEPT
+[id="nw-configuring-elb-timeouts-aws-classic_{context}"]
+= Configuring Classic Load Balancer timeouts on AWS
+
+{product-title} provides a method for setting a custom timeout period for a specific route or Ingress Controller. Additionally, an AWS Classic Load Balancer (CLB) has its own timeout period with a default time of 60 seconds.
+
+If the timeout period of the CLB is shorter than the route timeout or Ingress Controller timeout, then the load balancer can prematurely terminate the connection. You can mitigate this by increasing both the timeout period of the route and CLB.
```

Comment:
```suggestion
If the timeout period of the CLB is shorter than the route timeout or Ingress Controller timeout, the load balancer can prematurely terminate the connection. You can mitigate [alt: prevent?] this [problem?] by increasing both the timeout period of the route and CLB.
```
ISG: You can skip "then" if introducing an independent clause that follows an "if" clause.